datasets/attack_techniques/T1485/s3_bucket_deletion/s3_bucket_deletion.csv

```diff
@@ -0,0 +1,33 @@
col,"protocol_code",punct,readOnly,reason,recipientAccountId,region,requestID,"requestParameters.Host","requestParameters.WebsiteConfiguration.ErrorDocument.Key","requestParameters.WebsiteConfiguration.IndexDocument.Suffix","requestParameters.WebsiteConfiguration.xmlns","requestParameters.agentName","requestParameters.agentStatus","requestParameters.agentVersion","requestParameters.availabilityZone","requestParameters.availabilityZoneId","requestParameters.bucketName","requestParameters.bucketPolicy.Statement{}.Action","requestParameters.bucketPolicy.Statement{}.Effect","requestParameters.bucketPolicy.Statement{}.Principal","requestParameters.bucketPolicy.Statement{}.Resource","requestParameters.bucketPolicy.Statement{}.Sid","requestParameters.bucketPolicy.Version","requestParameters.computerName","requestParameters.durationSeconds","requestParameters.evaluations{}.complianceResourceId","requestParameters.evaluations{}.complianceResourceType","requestParameters.evaluations{}.complianceType","requestParameters.evaluations{}.orderingTimestamp","requestParameters.iPAddress","requestParameters.instanceId","requestParameters.location","requestParameters.platformName","requestParameters.platformType","requestParameters.platformVersion","requestParameters.policy","requestParameters.resultToken","requestParameters.roleArn","requestParameters.roleSessionName","requestParameters.sSMConnectionChannel","requestParameters.testMode","requestParameters.website","resources{}.ARN","resources{}.accountId","resources{}.type",responseElements,"responseElements.assumedRoleUser.arn","responseElements.assumedRoleUser.assumedRoleId","responseElements.credentials.accessKeyId","responseElements.credentials.expiration","responseElements.credentials.sessionToken",result,"result_id","rule_action",sharedEventID,signature,source,sourceIPAddress,sourcetype,"splunk_server","splunk_server_group",src,"src_ip","src_ip_range","src_port_range","src_user","src_user_id","src_user_name","src_user_role","src_user_type","start_time",status,tag,"tag::action","tag::app","tag::eventtype","tag::object_category","temp_access_key",timeendpos,timestartpos,"tlsDetails.cipherSuite","tlsDetails.clientProvidedHostHeader","tlsDetails.tlsVersion",user,userAgent,"userIdentity.accessKeyId","userIdentity.accountId","userIdentity.arn","userIdentity.invokedBy","userIdentity.principalId","userIdentity.sessionContext.attributes.creationDate","userIdentity.sessionContext.attributes.mfaAuthenticated","userIdentity.sessionContext.ec2RoleDelivery","userIdentity.sessionContext.sessionIssuer.accountId","userIdentity.sessionContext.sessionIssuer.arn","userIdentity.sessionContext.sessionIssuer.principalId","userIdentity.sessionContext.sessionIssuer.type","userIdentity.sessionContext.sessionIssuer.userName","userIdentity.type","userIdentity.userName",userName,"user_access_key","user_agent","user_arn","user_group_id","user_id","user_name","user_role","user_type",vendor,"vendor_account","vendor_product","vendor_region"
```

Comment:
@josehelps : We would need the Cloudtrail, DNS and  Web _raw logs for the attack data for testing. This looks like a csv